configure.ac

```diff
@@ -2133,6 +2133,17 @@ if test "${CHIP_DEVICE_LAYER_TARGET_LINUX}" = 1; then
         AC_DEFINE([CHIP_WITH_GIO], [0], [Define to 0 to build CHIP without GIO])
     ]
     )
+
+    PKG_CHECK_MODULES(GIOUNIX, gio-unix-2.0 >= 2.28, dummy=yes,
```

Comment:
yufengwangca 26 seconds ago Member

It there any specific reason we need gio-unix-2.0 >= 2.28? gio-2.0 and and gio-unix-2.0 should be version compatible. Can we just use 'PKG_CHECK_MODULES([GIO_UNIX], [gio-unix-2.0])'?

Add following defines in 'Configuration Summary'
GIO UNIX compile flags : ${GIO_UNIX_CFLAGS:--}
GIO UNIX link flags : ${GIO_UNIX_LIBS:--}

Reply:
Done, thanks.